File: core_backend/core_backend/Models/SensorGeolocation.cs
using System.ComponentModel.DataAnnotations;

namespace core_backend.Models
{
    public class SensorGeolocation
    {
        [Key]
        public string Uuid { get; set; }
        public float Latitude { get; set; }
        public float Longitude { get; set; }
        public float Altitude { get; set; }
        public float Bearing { get; set; }
        public float Accuracy { get; set; }
        public string SensoryType { get; set; }
        public string Provider { get; set; }
        public bool IsNoise { get; set; }
        public long CreatedOn { get; set; }
        public long DeletedOn { get; set; }
        public long BatteryLevel { get; set; }
        public string UserId { get; set; }
    }
}

File: backend/models/sensor_geolocation.go
package models

type SensorGeolocation struct {
	Uuid         string  `gorm:"primary_key" json:"uuid"`
	Latitude     float64 `json:"latitude"`
	Longitude    float64 `json:"longitude"`
	Altitude     float64 `json:"altitude"`
	Bearing      float64 `json:"bearing"`
	Accuracy     float64 `json:"accuracy"`
	Speed        float64 `json:"speed"`
	SensorType   string  `json:"sensorType"`
	Provider     string  `json:"provider"`
	IsNoise      bool    `json:"isNoise"`
	CreatedOn    int64   `json:"createdOn"`
	DeletedOn    int64   `json:"deletedOn"`
	BatteryLevel int64   `json:"batteryLevel"`
	UserId       int64
}
